File: Universal Windows Platform Solution/Phantom of the West/Phantom of the West/Visual Novel/Chapters/Chapter 9/ILeaveFoxtrot.cs
using Phantom_of_the_West.Visual_Novel.Serialization.Event_Flagging;

namespace Phantom_of_the_West.Visual_Novel.Chapters.Chapter_9
{
	internal class ILeaveFoxtrot : StoryChoice
	{
		internal ILeaveFoxtrot() : base(StoryChoice.GetLocalizedString("ch9ILeaveFoxtrot"), 9094)
		{
		}

		public override void Select()
		{
			PotWVN vn = PotWVN.MainVN;
			vn.EventFlags.Ch9FoxtrotReacted = FoxtrotReaction.NotPetted;
			base.Select();
		}
	}
}

File: Android Project/app/src/main/java/com/christopherraleigh/phantomofthewest/visual_novel/chapters/chapter_9/ILeaveFoxtrot.java
package com.christopherraleigh.phantomofthewest.visual_novel.chapters.chapter_9;

import com.christopherraleigh.phantomofthewest.R;
import com.christopherraleigh.phantomofthewest.visual_novel.PotWVN;
import com.christopherraleigh.phantomofthewest.visual_novel.chapters.StoryChoice;
import com.christopherraleigh.phantomofthewest.visual_novel.serialization.event_flagging.FoxtrotReaction;

/**
 * Created by christopherraleigh on 2016-05-21.
 */
public class ILeaveFoxtrot extends StoryChoice {
    public ILeaveFoxtrot() {
        super(R.string.ch9ILeaveFoxtrot, 9094);
    }

    @Override
    public void select()
    {
        PotWVN vn = PotWVN.getMainVN();
        vn.getEventFlags().setCh9FoxtrotReacted(FoxtrotReaction.NotPetted);
        super.select();
    }
}
